editor/editor_run.cpp

```diff
@@ -203,6 +203,13 @@ Error EditorRun::run(const String &p_scene, const String &p_write_movie) {
 		args.push_back("--skip-breakpoints");
 	}
 
+	String shortcut;
+	Ref<Shortcut> shrt = ED_GET_SHORTCUT("editor/stop")->duplicate();
+	shrt->set("__meta__", Variant()); // We don't need to send the metadata.
```

Comment:
This looks a bit unexpected at a glance. It might be an argument for adding a `clear_meta()` method that clears all metadata on an object.

Reply:
Ah, this will no longer work...
Yeah, a `clear_meta()` would be useful here.

Reply:
The shortcuts seem to no longer use metadata (or it's not serialized to the string), so this is not necessary anymore.